src/components/layout/layout.js

```diff
@@ -79,16 +80,55 @@ const RouteChangeScroller = () => {
   return null;
 };
 export function Layout({ children }) {
+
+  // query for banner frontmatter and body text
+  const data = useStaticQuery(graphql`
+    query {
+      activeBanners: allMdx(
+        sort: {frontmatter: {importance: ASC}}
+        filter: {
+          internal: {contentFilePath: {regex: "/src/data/banners/"}}, 
+          frontmatter: {active: {eq: true}}
+        }
+      ) {
+        nodes {
+          frontmatter {
+            variant
+            active
+          }
+          internal {
+            contentFilePath
+          }
+          body
+        }
+      }
+    }
+  `);
+
+  // transform incoming banner data into shape { variant, content }
+  const banners = data.activeBanners.nodes.map(n => ({
+    variant: n.frontmatter.variant,
+    content: n.body
+  }))
+
   const { isCompact } = useWindowWidth();
 
   return (
     typeof isCompact === "boolean" && (
       <LayoutWrapper compact={isCompact ? true : undefined}>
         <RouteChangeScroller />
         <SkipLink href="#main-content">Skip to main content</SkipLink>
-        <Banner variant="info">
-          NHLBI BioData Catalyst® (BDC) supports data and analysis in a secure, FISMA-moderate environment.<br/>BDC security controls adhere to <Link to="https://grants.nih.gov/grants/guide/notice-files/NOT-OD-24-157.html">NIH's Implementation Update for Data Management and Access Practices Under the Genomic Data Sharing Policy (NOT-OD-24-157)</Link>.
-        </Banner>
+        {
+          // check if there are any active banners; only render if active banners exist
+          (banners.length > 0) && (
```

Comment:
a couple things here:
- don't forget keys in this map
- we don't really need to do this "check if there are any active banners" here. if no banners are active, this is a map over an empty array, which yields an empty array, and React can handle that. so this could be reduced to something like this.
```
        { // render active banners
          banners.map((banner, i) => (
            <Banner
              key={`banner-{i}`}
              variant={banner.variant} 
              children={banner.content}
            />
          )) }
```